src/test/java/LoginTests.java

```diff
@@ -88,4 +93,5 @@ public static void RegistrationLinkNavigationTest() throws InterruptedException
 
 
     }
+
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210047663-737f5d0a-a41a-4445-a576-5ea26d7c6294.png)